### PR TITLE
Issue 1416 how to MQTT Kafka source connector

### PR DESCRIPTION
**Pull request by jorgeuribe-aiven**
# What changed, and why it matters
I have create a howto document to show how to configure MQTT Kafka source connector and configure it to get messages from RabbitMQ/MQTT servers.

Fixes https://github.com/aiven/devportal/issues/1416

### CODE REVIEW

File: docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
+Create a MQTT source connector for RabbitMQ®
+============================================
+
+In order to support the `MQTT source connector <https://docs.lenses.io/5.0/integrations/connectors/stream-reactor/sources/mqttsourceconnector/>`_ the `RabbitMQ MQTT plugin <https://www.rabbitmq.com/mqtt.html>`_ must be enabled.
```
Comment (ftisiot) [inline docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst:4]: I would start with a phrase on what the connector does before jumping to the details of the plugin

File: docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
+Create a MQTT source connector for RabbitMQ®
```
Comment (ftisiot) [inline docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst:1]: Can we call this `Create a MQTT source connector` and then just say in the text that we're taking the RabbitMQ eexample

File: docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+    The Aiven Console parses the configuration file and fills the relevant UI fields. You can review the UI fields across the various tab and change them if necessary. The changes will be reflected in JSON format in the **Connector configuration** text box.
+
+7. After all the settings are correctly configured, click on **Create new connector**
+8. Verify the connector status under the **Connectors** tab
+9. Verify the presence of the data in the target Apache Kafka topic, the topic name is the one defined in the ``KCQL_STATEMENT``
+
+Example: Create a MQTT source connector from Aiven CLI
+------------------------------------------------------
+The Connectors can be created also using the dedicated :ref:`Aiven CLI command <avn_service_connector_create>`
```
Comment (ftisiot) [inline docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst:87]: Let's remove the CLI part, we had the agreement in the past of only showing the Console example

Comment (ftisiot) [inline docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst:87]: We can however add this sentence to the end of the Console example with a `Tip`

**Review by ftisiot — CHANGES_REQUESTED** [review]
Thanks @jorgeuribe-aiven 

Minimal changes needed. The CLI section is not needed

File: docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst (hunk 1 above)
Comment (jorgeuribe-aiven) [reply docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst]: done

File: docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst (hunk 2 above)
Comment (jorgeuribe-aiven) [reply docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst]: done

File: docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst (hunk 3 above)
Comment (jorgeuribe-aiven) [reply docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst]: section removed

Comment (jorgeuribe-aiven) [reply docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst]: Tip added

**Comment by jorgeuribe-aiven** [comment]
@ftisiot I have pushed a new commit with the changes you suggested.

**Review by ftisiot — APPROVED** [review]
LGTM! Thanks @jorgeuribe-aiven!

**Pull request merged**